src/ajax/xhr.js

```diff
@@ -88,12 +88,15 @@ jQuery.ajaxTransport( function( options ) {
 									xhrSuccessStatus[ xhr.status ] || xhr.status,
 									xhr.statusText,
 
-									// Support: IE9
```

Comment:
I removed this comment because basically all XHR2 impls are supposed to throw on binary data in cases like ArrayBuffer.